src/functions_dims.jl

```diff
@@ -21,10 +21,15 @@ end
 function Base.permutedims(nda::NamedDimsArray{L}, perm) where {L}
     numerical_perm = dim(nda, perm)
     new_names = permute_dimnames(L, numerical_perm)
-
     return NamedDimsArray{new_names}(permutedims(parent(nda), numerical_perm))
 end
 
+function Base.PermutedDimsArray(nda::NamedDimsArray{L}, perm) where {L}
+    numerical_perm = dim(nda, perm)
+    new_names = permute_dimnames(L, numerical_perm)
+    return NamedDimsArray{new_names}(PermutedDimsArray(parent(nda), numerical_perm))
+end
+
```

Comment:
I lean towards writing this together with `permutedims` as


```
for f in (
    :(Base.permutedims),
    :(Base.PermutedDimsArray)
)
    @eval function $f(nda::NamedDimsArray{L}, perm) where {L}
        numerical_perm = dim(nda, perm)
        new_names = permute_dimnames(L, numerical_perm)
        return NamedDimsArray{new_names}($f(parent(nda), numerical_perm))
    end
end
```

Reply:
Sure, done!